rules/eslint/stylistic-issues/on.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
         'one-var-declaration-per-line': 0,
         'one-var': [2, { var: 'always', let: 'never', const: 'never' }],
         'operator-assignment': 0,
-        'operator-linebreak': 0,
+        'operator-linebreak': ['error', 'after'],
```

Comment:
Ideal this should confirm to the other rules and use integers to denote the error type [like so](http://eslint.org/docs/user-guide/configuring#configuring-rules): `'operator-linebreak': [2, 'after'],`